pages/health-care/eligibility/active-duty.md

```diff
@@ -0,0 +1,146 @@
+[Find out how to apply for VA health care](/health-care/apply)
+
+------
+<span id="separating"></span>
+### If you're separating from active duty
+
+You may be eligible for VA health care for all or some of your health care needs. We'll determine your eligibility based on your service history, VA disability rating, income, and other factors. <br>
+[Learn more about VA health care eligibility](/health-care/eligibility) <br>
+[Find out how to apply for VA health care](/health-care/apply)
+
+You, and your family, may also qualify for temporary health coverage through the Department of Defense's TRICARE program. If you're a member of the National Guard and Reserve, your coverage options will depend on your military status before you separate. You'll have 90 days from your separation date to change your health plan.
+
+Learn more about how to enroll for temporary coverage after separation. <br>
+[Go to the TRICARE website page on separating](https://tricare.mil/LifeEvents/Separating)
+
+------
+<span id="retiring"></span>
+### If you're retiring from active duty
+
+You may be eligible for the Department of Defense's TRICARE program as a military retiree. Your family members may also qualify for care. You'll need to enroll within 90 after your retirement date even if you're already enrolled in TRICARE on active duty.
+
+Learn more about how to enroll in TRICARE as a military retiree. <br>
+[Go to the TRICARE website page on retiring](https://tricare.mil/LifeEvents/Retiring)
+
+You may also be eligible for certain VA health care benefits. <br>
+[Learn more about VA health care eligibility](/health-care/eligibility) <br>
+[Find out how to apply for VA health care](/health-care/apply)
+
+------
+<span id="medically-retired"></span>
+### If you're being medically retired from active duty
+
+You may be eligible to receive care for any service-connected disabilities through VA. You may also qualify to receive all other care through TRICARE. Your eligible family members may also qualify for care through TRICARE. 
+
+To learn more about your options, talk with a VA benefits counselor. Your counselor will help you determine what your military and VA medical board ratings mean and how they affect your options. Your counselor will also help you apply for benefits.
+
+Call our health benefits hotline at 1-877-222-VETS (<a href="tel:+18778387">1-877-8387</a>) to connect with a benefits counselor.  If you have hearing loss, call TTY: 1-800-877-8339. We're Monday through Friday, 8:00 a.m. to 8:00 p.m. ET. Or contact your nearest VA medical center. <br>
+[Find a VA medical center near you](/find-locations/?zoomLevel=4&page=1&address=&facilityType=health&serviceType)
+
+<script type="text/javascript">
+  // Toggle the expandable crisis info
+  document.getElementById('crisis-expander-link')
+    .addEventListener('click', function () {
+      document.getElementById('crisis-expander-content').classList.toggle('expander-content-closed');
+    });
+</script>
```

Comment:
This script is used to expand content for Crisis Line alerts, but needs to be updated for the Mental Health alerts. Would you like me to update this?

Reply:
Yes, please - thank you!